libnopegl/src/node_drawpath.c

```diff
@@ -303,8 +292,7 @@ static int drawpath_init(struct ngl_node *node)
     s->projection_matrix_index = ngpu_pgcraft_get_uniform_index(s->crafter, "projection_matrix", NGPU_PROGRAM_STAGE_VERT);
     s->transform_index         = ngpu_pgcraft_get_uniform_index(s->crafter, "transform", NGPU_PROGRAM_STAGE_VERT);
 
-    s->coords_fill_index    = ngpu_pgcraft_get_uniform_index(s->crafter, "coords_fill", NGPU_PROGRAM_STAGE_FRAG);
-    s->coords_outline_index = ngpu_pgcraft_get_uniform_index(s->crafter, "coords_outline", NGPU_PROGRAM_STAGE_FRAG);
+    s->coords_index = ngpu_pgcraft_get_uniform_index(s->crafter, "coords", NGPU_PROGRAM_STAGE_FRAG);
```

Comment:
nit: alignment.

Reply:
I didn't bother with it because it was a separate block (vert vs frag block), but just changed it to be all packed in a single block.